.github/workflows/build.yml

```diff
@@ -60,9 +60,13 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-go-
     - name: Run e2e tests
-      run: make crd e2e_test -e CRD_SPEC_VERSION=${{ matrix.crd-spec-version }} -e KIND_NODE_VERSION=${{ matrix.kind-node-version }} -e KIND_KUBECTL_ARGS=--validate=false
-    - name: Show e2e debug logs
-      run: cat e2e/debug/detik/*
+      run: make crd e2e-test -e CRD_SPEC_VERSION=${{ matrix.crd-spec-version }} -e KIND_NODE_VERSION=${{ matrix.kind-node-version }} -e KIND_KUBECTL_ARGS=--validate=false -e bats_args="-F junit || true"
```

Comment:
Had to append `bats_args="-F junit || true"` to the bats command in order to be able to publish test reports. Bats would otherwise exit on errors without having a chance to upload the reports because GH workflow cancels on failures.

Any better solutions?

Reply:
Just for clarification: the `|| true` is there in case bats exits with errors, but still has something to upload?

Reply:
yes, exactly

Reply:
If there's no flag to catch it in bats directly I'd say that's the way to go.